lib/runtime/Cargo.toml

```diff
@@ -41,6 +41,7 @@ singlepass = ["wasmer-singlepass-backend"]
 default-backend-singlepass = ["singlepass"]
 default-backend-llvm = ["llvm"]
 default-backend-cranelift = ["cranelift"]
+deterministic = ["wasmer-singlepass-backend/deterministic"]
```

Comment:
This should probably trigger `wasmer-runtime-core/deterministic`, too

Reply:
I thought it's redundant because:
https://github.com/wasmerio/wasmer/blob/master/lib/singlepass-backend/Cargo.toml#L12

Reply:
It's redundant in that all interactions with wasmer-runtime-core through singlepass will be deterministic as defined by that flag, but I believe Rust will compile a separate version of the crate with the other feature flags that are directly accessible from wasmer-runtime